z_json_test.go

```diff
@@ -705,6 +705,126 @@ func TestUpdateJSONScalar(t *testing.T) {
 	}
 }
 
+// Converts map with different Go-types to godror.JSON type
+// This is bound to the insert function and executed
+// For each unique go-type in the map, their corresponding JSON types,
+// as stored in the DB, are fetched and compared with their expected values.
+func TestStorageTypes(t *testing.T) {
```

Comment:
Use a more specific name: TestJSONStorageTypes as this test needs a high enough DB version.